addons/binding/org.openhab.binding.openuv/README.md

```diff
@@ -1,10 +1,11 @@
-# Open UV Binding
+# OpenUV Binding
 
 This binding uses the [OpenUV Index API service](https://www.openuv.io/) for providing UV Index information for any location worldwide.
 
 To use this binding, you first need to [register and get your API token](https://www.openuv.io/auth/google).
 
-## Supported Things
+## Binding Installation
```

Comment:
Please add an empty line after the headline and put each sentence I to a separate line.

Reply:
ok done